CONTRIBUTING.md

```diff
@@ -93,28 +87,28 @@ Since the deploy preview does not work on Draft PRs, follow these steps to previ
 
 1. If you have not already, clone the tutorials repo to your local machine and change to that directory. For example:
 
-    ```bash
-    cd ~/github
-    git clone --recursive https://github.com/cosmos/sdk-tutorials
-    cd sdk-tutorials
-    ```
+   ```sh
+   cd ~/github
+   git clone --recursive https://github.com/cosmos/sdk-tutorials
+   cd sdk-tutorials
+   ```
```

Comment:
I am not sure why change the indentation from 4 spaces to 3.

Reply:
It is formatting issue an linter cursed.

Reply:
Your other changes are valid.